frontend/src/components/worksheets/WorksheetNameSearch.js

```diff
@@ -19,8 +19,9 @@ export default class extends React.Component {
         }
 
         try {
-            const response = await fetch(`/rest/worksheets?specs=${name}`).then((e) => e.json());
-            const uuid = response.data[0].id;
+            const response = await fetch(`/rest/worksheets?specs=${name}`);
```

Comment:
I think in this case, we don't necessarily need to add the additional `await` here. Personally, `.then(e => e.json())` seems simpler / more readable than having to declare a new variable called `json`.